src/Templates/Webcam/focus.twig

```diff
@@ -1,15 +1,17 @@
 {% extends 'stripe.twig' %}
 
 {% block stripecontent %}
+<link rel="stylesheet" type="text/css" href="{{baseurl}}css/myradio.webcamfocus.css">
```

Comment:
This won't work, because `stripecontent` is within `<body>`. You'll need to move this to `{% block head %}` (I think could be wrong).

Reply:
Tested locally and it works bcs HTML👏 but you're right does need to go above.